models/channel/src/channel-draft/presets/index.ts

```diff
@@ -1,4 +1,4 @@
-import * as SampleDataFashion from './sample-data-fashion';
```

Comment:
Why did you remove the `* as`?

Reply:
When requiring in from other packages, `* as` had the following effect:

```
import {draftPresets, presets} from '@commercetools-test-data/channel';

presets.clothesStore

draftPresets.SampleDataFashion.default.storeOther // with `* as`
draftPresets.SampleDataFashion.storeOther // without `* as`
```

`channel-draft.presets.index.ts` already has a `default` export, so the `* as` syntax is superfluous. [These docs ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#namespace_import)helped me understand what was going on here, too

Reply:
Ah! The zone presets need to be updated then because they use the `* as` syntax.

Reply:
Done! Addressed in 1b506a6e9f118d2e6614cc066b244fc44c9e1d2b